### DISCUSSION TITLE
about/

### DISCUSSION THREAD

**Discussion by giscus** [nickxudotme/nickxudotme.github.io#9] [General]
# about/

欢迎来到我的小站呀，很高兴遇见你！🤝  🏠 关于本站 这个独立博客网站是我在 2021 年创建的，当时是用 Gridea 搭建的。现在由于体验不佳而转投了 Hexo 框架+ Butterfly 主题。在改了几天配置文件后，今天（2022年2月7日）正式切换了新皮肤（真的好好看啊，为国人大佬

https://nickxu.me/about/

**Comment by AuggieChen** [comment]
大佬，在Hexo使用hexo d部署到github库后，代码库里的.github文件夹就消失了，里面用于github action的文件就消失了，同样github pages就不会自动更新，这怎么解决，求解！

**Reply by nickxudotme** [reply]
帮助解决了，手动往 `.deploy_git` 里加入然后再推送即可

**Comment by ZhangStudyLife** [comment]
大佬，我在查略hexo与butterfly如何评论的时候，发现了您的博客  
发现您也是HDU的学生，我是准大一新生，感叹于大佬您的厉害，自己还在研究怎么部署网站  
于此赞叹

**Reply by nickxudotme** [reply]
哦，谢谢你❤️
人外有人，天外有天，早点看见差距，就能早点开始进步👀

**Comment by Pan-O** [comment]
大佬文章写的都很不错，想订阅rss定期看看，但是rss好像设防盗链了🧐

**Reply by nickxudotme** [reply]
啊？我会看看👀

**Reply by nickxudotme** [reply]
```
curl -I https://nickxu.me/rss2.xml
HTTP/1.1 200 Connection established

HTTP/2 200
date: Sun, 29 Sep 2024 12:17:56 GMT
content-type: application/xml
access-control-allow-origin: *
cache-control: public, max-age=0, must-revalidate
etag: W/"7fa466a5a5738d8bc1d6bfff02bdc6c8"
referrer-policy: strict-origin-when-cross-origin
x-content-type-options: nosniff
report-to: {"endpoints":[{"url":"https:\/\/a.nel.cloudflare.com\/report\/v4?s=miA7G%2B5arUcBBX3JMmNXIPx3%2FWUV%2BkG6a1u7QpaJLhvcE4o2dnruVEvV%2BPMjreRWczeVSH0nq2XyMADRXxqx9ZJUtFZ1qkONXJuPdrXlhphyKrEieFk7a0btKZM%3D"}],"group":"cf-nel","max_age":604800}
nel: {"success_fraction":0,"report_to":"cf-nel","max_age":604800}
vary: Accept-Encoding
cf-cache-status: DYNAMIC
server: cloudflare
cf-ray: 8cabe9b6bd7cb470-HKG
```
我好像没有看见有什么问题？👀

**Reply by Pan-O** [reply]
我使用本地的rss添加的话也没有问题，但是当我使用部署在服务器上的 `Miniflux` rss服务器去订阅的话会返回403，然后当我使用代理服务器去代理 `https://nickxu.me/rss2.xml` 这个地址就可以正常订阅，我的代理服务器只是伪装了referrer的请求头，所以我感觉可能是设置了防盗链🤔
